seqr/views/utils/orm_to_json_utils.py

```diff
@@ -112,8 +112,10 @@ def _get_json_for_user(user):
         user = user._wrapped   # Django request.user actually stores the Django User objects in a ._wrapped attribute
 
     user_json = {_to_camel_case(field): getattr(user, field) for field in
-                ['username', 'email', 'first_name', 'last_name', 'last_login', 'is_staff', 'date_joined', 'id']}
+                ['username', 'email', 'first_name', 'last_name', 'last_login', 'is_staff', 'date_joined', 'id',
+                 'assigned_families']}
```

Comment:
I'm not sure we need to return `assigned_families` at all. Because we have the `assigned_to` value on the family itself this actually becomes a lot of redundant data. We should discuss this more offline